openassessment/xblock/openassessmentblock.py

```diff
@@ -246,6 +247,25 @@ class OpenAssessmentBlock(MessageMixin,
         help="Indicates whether or not there are peers to grade."
     )
 
+    label_list = List(
+        default = DEFAULT_LABEL_LIST,
+        scope = Scope.content,
+        help="List of strings relavent to the content."
+    )
+
+    @property
+    def labels(self):
+        return ",".join(self.label_list)
+
+    @labels.setter
+    def labels(self, value):
+        """Removes duplicates and stores label list."""
```

Comment:
Use multi-line docstring(not a standard yet but it is easy to read).

Reply:
Done.